extras/dummy_generator/generator.py

```diff
@@ -452,10 +448,10 @@
 
             creation_date = datetime.date.today() - datetime.timedelta(days=i)
             if creation_date not in existing_entries:
-                entry = WeightEntry(user=user,
-                                    weight=args.base_weight + 0.5 *
-                                    i + random.randint(1, 3),
-                                    date=creation_date)
+                entry = WeightEntry(
+                    user=user,
+                    weight=args.base_weight + 0.5 * i + random.randint(1, 3),
```

Comment:
It's cleaner to have `weight` defined  outside like 

```python 
weight = args.base_weight + 0.5 * i + random.randint(1, 3)
```

Then have entry equal

```python
entry = WeightEntry(
    user=user,
    weight=weight,
    date=creation_date
)
```